blog/blocks/page-content/page-content.css

```diff
@@ -0,0 +1,5 @@
+.page-content {
+    max-width: 1152px;
```

Comment:
I think  that part can be simplified it's not so necessary to add that max-width I think  we can add there 858px  and remove from inner content that max-width